packages/trader/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -113,33 +98,14 @@ class RealAccountSignup extends Component {
                     ),
                 },
             ],
-            modal_height: ModalHeights.change_currency,
         };
     }
 
-    componentDidMount() {
-        const {
-            available_crypto_currencies,
-            can_change_fiat_currency,
-            currency,
-            has_real_account,
-        } = this.props;
-
-        let height = this.state.height;
-        if (!has_real_account) {
-            height = ModalHeights.account_wizard[0];
-        } else if (!currency) {
-            height = ModalHeights.set_currency;
-        } else if (available_crypto_currencies.length !== 0 && can_change_fiat_currency) {
-            height = ModalHeights.add_manage_account;
-        } else {
-            height = ModalHeights.add_currency;
-        }
-        this.setState({ modal_height: height });
-    }
-
-    setModalHeight = (step) => {
-        this.setState({ modal_height: ModalHeights.account_wizard[step] });
+    get modal_height() {
+        if (!this.props.currency) return '688px';
```

Comment:
Would you please add a comment here explaining the conditions and why we are setting different values? e.g.: `Currency set modal`